doc/075_scripting.rst

```diff
@@ -83,13 +83,23 @@ JSON output of most restic commands are documented here.
     list of allowed values is documented may be extended at any time.
 
 
+Exit errors
+-----------
+
+Fatal errors will result in a final JSON message before the process exits.
+It will hold the error message and the exit code.
+
++----------------------+-------------------------------------------+
+| ``message_type``     | Always "exit_error"                       |
++----------------------+-------------------------------------------+
+| ``code``             | Exit code (see above chart)               |
++----------------------+-------------------------------------------+
+| ``error``            | Error message                             |
++----------------------+-------------------------------------------+
+
 Output formats
 --------------
 
-Currently only the output on ``stdout`` is JSON formatted. Errors printed on ``stderr``
```

Comment:
I removed it because I felt like the errors-as-JSON situation was so much better that it didn't deserve a big warning. But how about the current text:

```

Commands print their main JSON output on ``stdout``.
The generated JSON output uses one of the following two formats.

.. note::
    Not all messages and errors have been converted to JSON yet.
    Feel free to submit a pull request!
```

That note mirrors an earlier one in the doc about some messages not supporting `--json` at all.

(I also emphasized that some error messages print to `stderr` a few more places in the docs.)